### DISCUSSION TITLE
Platzhalter in RIC

### DISCUSSION THREAD

**Discussion by lestho** [Schrolli91/BOSWatch#535] [General]
Hallo!
Gibt es in BosWatch einen Platzhalter für die RIC,  Benötige ich in mehrern Bereichen:
Sprich, ich habe die Filtertabelle 010000-099999 freigegeben, möchte aber den Block 050000-059999 Sperren, Gibts da was wie z.B. dass ich 05****** in die "Blocked RIC" eingebe und der Block gesperrt ist?
das selbe bräuchte ich für die "poc.template.csv", dort sollte es so aussehen: 0173***, "Musterland1" also Jede Ric was mit 0173 beginnt soll den Text "Musterland1" bekommen

Danke =)

**Comment by Schrolli91** [comment]
Also das Filtern der RICS zur Alarmierung ginge über den Regex Filter
https://github.com/Schrolli91/BOSWatch/blob/5ebbaba6f15b1e0fc0e01b26cfeaf22ebaf82252/config/config.template.ini#L151

Bei den Description Files bin ich mir grade nicht sicher. Aber ich glaube aus dem Stand ist das so momentan nicht möglich.
Mit BW3 wäre das evtl. einfach umzusetzen.
